tests/OK/prv.lp

```diff
@@ -0,0 +1,3 @@
+symbol const A : TYPE
+
+symbol private f : A ⇒ A
```

Comment:
I don't like this. private is not the same kind of property as const or injective. It's about scoping. We should write: `private symbol f:A ⇒ A`. The semantics is: f is not exported.

Reply:
Regarding the syntax: this was the easiest way; but I agree it is arguable.

Reply:
And what about using a `set` command? Like `set f private`?

Reply:
I think it is best to keep it as it is now. It might make sense to combine several modifiers and the syntax is more uniform that way.

Reply:
I do not agree. Private could be applied to other things than just symbols.

Reply:
Like private rules?

Reply:
Why not. I was thinking in private tactics, notations, etc.